pom.xml

```diff
@@ -9,20 +9,53 @@
 	<packaging>pom</packaging>
 
 	<properties>
+		<java.version>11</java.version>
+		<maven.compiler.source>${java.version}</maven.compiler.source>
```

Comment:
Doesn't the maven.compiler.release contain/replace the compiler.source and compiler.target? https://www.baeldung.com/maven-java-version#java9

Reply:
removed target and source then